quickstarts/gradle/groovy-dsl-config/gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
```

Try to use the same Gradle distribution for all quick starts:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.1.1-bin.zip
```